website/app/components/doc/component-api/property.hbs

```diff
@@ -12,6 +12,9 @@
     {{#if @required}}
       <code class="doc-component-api__property-required">required</code>
     {{/if}}
+    {{#if @deprecated}}
+      <Doc::Badge class="doc-component-api__property-deprecated" @type="warning" @size="medium">Deprecated</Doc::Badge>
```

Comment:
perhaps more of a design question: should we align the 'required' (all lowercase) with 'Deprecated' (starts with capital letter)

Reply:
@majedelass @heatherlarsen ?

Reply:
@didoo Is there an example of one that includes both?

Reply:
I've updated the example in the temporary page to have both: https://hds-website-git-website-update-frontmatter-add-e2df27-hashicorp.vercel.app/components/accordion?tab=code#aaccordionitem

They look like this:
![image](https://github.com/hashicorp/design-system/assets/686239/a60d3e04-8dc9-490a-a5b9-5369142270fc)

Reply:
I honestly wouldn't mind moving the "required" into a badge, but I think this is fine for now and we can see if there's any feedback about it later.

Reply:
@heatherlarsen this is how it would look like
![image](https://github.com/hashicorp/design-system/assets/686239/58c1c6c8-22a6-4af4-b422-fac4e23d7d6c)

you can see it in this demo page too:
https://hds-website-git-website-update-frontmatter-add-e2df27-hashicorp.vercel.app/components/accordion?tab=code#contextual-components

one consideration: not very likely that an argument/prop is at the same time required _and_ deprecated, so at the end of the day the question is if you prefer the "required" as a badge or as normal styled text (personally, I prefer the second, the badge is a bit too much in the face)

let me know if you want me to make the make the change or leave it as it was before (styled text, not badge)

Reply:
Based on a [quick poll](https://hashicorp.slack.com/archives/C025N5V4PFZ/p1720719564926969) we'd like to move forward with using the inverted critical badge for the required indicators.

Reply:
Love it!